packages/dashboard/src/index.ts

```diff
@@ -1,26 +1,28 @@
+import { Backend, createBackendFromDriver } from "@sidequest/backend";
 import { logger } from "@sidequest/core";
 import express from "express";
 import basicAuth from "express-basic-auth";
 import expressLayouts from "express-ejs-layouts";
 import morgan from "morgan";
 import { Server } from "node:http";
 import path from "node:path";
-import { initBackend } from "./backend-driver";
 import { DashboardConfig } from "./config";
-import dashboardRouter from "./resources/dashboard";
-import jobsRouter from "./resources/jobs";
-import queuesRouter from "./resources/queues";
+import { createDashboardRouter } from "./resources/dashboard";
+import { createJobsRouter } from "./resources/jobs";
+import { createQueuesRouter } from "./resources/queues";
 
 export class SidequestDashboard {
-  private app: express.Express;
+  private app?: express.Express;
   private config?: DashboardConfig;
   private server?: Server;
+  private backend?: Backend;
 
   constructor() {
     this.app = express();
   }
 
   async start(config?: DashboardConfig) {
+    this.app ??= express();
```

Comment:
since we have this on constructor:
```js
  constructor() {
    this.app = express();
  }
```

Is it possible to be undefined on start?

Reply:
We can make it undefined. I will do it.